addons/mail/static/tests/discuss_app/discuss.test.js

```diff
@@ -1841,7 +1841,7 @@ test("Correct breadcrumb when open discuss from chat window then see settings",
     await click("[title='Channel settings']", {
         parent: [".o-mail-DiscussSidebarChannel", { text: "General" }],
     });
-    await contains(".o_breadcrumb", { text: "DiscussGeneral" });
+    await contains(".o_breadcrumb", { text: "General" });
```

Comment:
There should be 2 times General, the one from the form view and one in the breadcrumb to go back to the channel itself.